.gitignore

```diff
@@ -0,0 +1,30 @@
+dev-practices/_build/doctrees/environment.pickle
```

Comment:
This file should go in the `dev-practises` directory, and probably needs to contain only the line:
```
_build
```
See the other doc directories for examples.

Reply:
I had a look at your ``simulatorUser`` and ``pipelineUser`` directories and it seems there is the same setup (in that the pickle file is in the ``doctrees`` subfolder). Have I misunderstood something?

Reply:
I'm talking about the `.gitignore` file.

Reply:
Sorry I missed that. Fixed.